protostar/configuration_file/configuration_file_v2.py

```diff
@@ -72,6 +71,9 @@ def get_declared_protostar_version(self) -> Optional[ProtostarVersion]:
             return None
         return parse_protostar_version(version_str)
 
+    def get_filepath(self) -> Path:
+        return self._file_path
```

Comment:
```suggestion
    def get_filepath(self) -> Path:
        return self._file_path
```
Why don't make this property public? Imo it is unnecessary biolderplate

Reply:
`ConfigurationFile` callers communicate with concrete classes through interface. `get_filepath` needs to be defined in that interface.